x-pack/plugins/spaces/server/spaces_client/spaces_client.ts

```diff
@@ -173,7 +173,17 @@ export class SpacesClient implements ISpacesClient {
   private transformSavedObjectToSpace(savedObject: SavedObject<any>) {
     return {
       id: savedObject.id,
-      ...savedObject.attributes,
-    } as Space;
+      name: savedObject.attributes.name,
```

Comment:
Should we have additional checks here to validate the presence of these attributes?

Reply:
I think it's a good idea; respective fields could be added to the `savedObjects` and `expectedSpaces` data in the test file.

Reply:
I've made the conversion in both directions explicit, and have updated the tests accordingly. Thanks for the feedback!